dask/dataframe/core.py

```diff
@@ -4123,6 +4123,12 @@ def to_timedelta(arg, unit='ns', errors='raise'):
                           meta=meta)
 
 
+@wraps(pd.isna)
```

Comment:
Since this function is only in newer versions of pandas, you'll have to wrap both the definition and the import in `__init__.py` in a case statement. I'd do something like:


```python
if hasattr(pd, 'isna'):
    @wraps(pd.isna)
    def isna(arg):
        return map_partitions(pd.isna, arg)
```

And in `__init__.py`:

```python
try:
    from .core import isna
except ImportError:
    pass
```